doc/source/workflows/events.rst

```diff
@@ -38,6 +39,47 @@ Workflow events are a special type of workflow task. They "finish" when the even
     workflow.run(gather.bind(event1_task, event2_task))
 
 
+HTTP Events
+-----------
+
+:ref:`Ray Serve <rayserve>` enables receiving workflow events via HTTP.
+To use HTTP events, we will install an HTTP event listener in the workflow and use Ray Serve to expose an HTTP endpoint. Below is an end-to-end example for installing the endpoint and receiving an HTTP event in a workflow.
```

Comment:
We don't need to mention serve here since this is purely internal implementation.

Reply:
Done. All mentioning of Serve are removed.